test/cache.test.js

```diff
@@ -288,7 +300,8 @@ test.cb("should allow to specify the .babelrc file", t => {
             exclude: /node_modules/,
             query: {
               cacheDirectory: t.context.cacheDirectory,
-              babelrc: path.join(__dirname, "fixtures/babelrc"),
+              extends: path.join(__dirname, "fixtures/babelrc"),
+              babelrc: false,
```

Comment:
This is the breaking change basically, I couldn#t figure out why we used a string value for `babelrc` in the first place.